src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -269,6 +269,13 @@ private void updateInstance(Instance instance, RoutingContext rContext, WebConte
   private boolean areInstanceBlockedFieldsChanged(Instance existingInstance, Instance updatedInstance) {
     JsonObject existingInstanceJson = JsonObject.mapFrom(existingInstance);
     JsonObject updatedInstanceJson = JsonObject.mapFrom(updatedInstance);
+
+    // We still need zeroing "succeedingInstanceId" in precedingTitles and "precedingInstanceId" in succeedingTitles
+    // because these fields are not provided for/from UI requests. We just ignore these fields on comparing as a blocked fields.
+    // Anyway they are filled after this comparing while fetching from DB in "updatePrecedingSucceedingTitles"-method.
+    zeroingField(existingInstanceJson.getJsonArray(Instance.PRECEDING_TITLES_KEY), PrecedingSucceedingTitle.SUCCEEDING_INSTANCE_ID_KEY);
+    zeroingField(existingInstanceJson.getJsonArray(Instance.SUCCEEDING_TITLES_KEY), PrecedingSucceedingTitle.PRECEDING_INSTANCE_ID_KEY);
```

Comment:
Why are they not provided by the UI?

Is it due to the representation of these structures being different in the business logic API (because those `id` properties are always the `id` of the current instance?

Reply:
Hi @marcjohnson-kint !

I think UI doesn't have it because 'mod-inventory' returns this structure even if we just search Instance in "inventory" app:

```
  "precedingTitles" : [ {
    "id" : "b26d82fa-e648-46c4-be47-ea1b4db32333",
    "title" : "Houston oil directory",
    "identifiers" : [ ]
  } ],
```